data/tasklist.txt

```diff
@@ -0,0 +1,7 @@
+This is your current task list:
+1. [E][ ] cry         (From: 2pm        To: 8am) 
```

Comment:
Maybe you could include the date along with the time? Since in your code  for deadline and event you are using LocalDateTime